UXPin/uxpin-merge-tools
# Handle `React.ReactChild` correctly

## Goal

Altassian Design system  uses `React.ReactChild` for some properties, like the `iconAfter` for Button: https://atlassian.design/components/button/examples

Currently it does not show up correctly in Merge, a yellow warning is displayed because the type is serialized incorrectly as `any`,

This PR fixes the behavior to handle `React.ReactChild` as if it was a `React.ReactNode`, which is the usual way to handle React nodes.

## How to test

Run experimental mode from Altassian DS repo: https://github.com/uxpin-merge/atlas-uxpin-merge/blob/main/src/components/Button/Button.tsx

Check that both `children` and `icon` fields are displayed correctly

## Screenshot

![image](https://github.com/UXPin/uxpin-merge-tools/assets/5546996/91557de1-e08e-4dd2-affc-62c21f025dc4)
